sesje/2015-02-11/script5.md

```diff
@@ -0,0 +1,46 @@
+# Zadanie 5
+
+Czyszczenie konsoli
+
+## Opis:
+
+Nasz gra strasznie śmieci w terminalu, ale możemy to zmienić. Za każdym razem gdy będziemy rozpoczynać nową rozgrywkę chcemy mieć pusty ekran z powitaniem.
+Możemy wykorzystać do tego moduł `os` i jego funkcję `system` by wywołać polecenie systemowe `cls` (dla linuxa i macosx `clear`).
+
+Przykład:
+```python
+import os
+
+os.system('cls')
```

Comment:
W zasadzie potrzebne jest tylko przypomnienie, że wciśnięcie F5 w IDLE, uruchomienie w CLI `python skrypt.py` albo samego `python` i wklejenie tam kodu to trzy różne rzeczy i mogą nie działać do końca identycznie.